README.md

```diff
@@ -176,22 +176,24 @@ And what is this ownership worth?
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
   may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
   under the laws of a specific state.
-
+- 🄳 A **startup** is an emerging company that aspires to grow quickly, especially by
+  providing something-like a product or technology-that is new to a market and has the
+  potential to transform that market.
+  Once a company is established in the market and successful for a while, it usually stops
```

Comment:
These time references are vague. What does "emerging" actually mean, wrt time or size? What measurements (time? size? money? 'presence'? office space? name recognition?) define "successful" and "for a while" and "established"? Offer some constraints/specifics.

Reply:
Also, what is "trade"

Reply:
I see that you've used the language from *Raising Venture Capital*. One dominant argument ([from Paul Graham](http://www.paulgraham.com/growth.html)) is that a startup is "a company designed to grow fast." Maybe start with what a startup *is not*. A startup is not:
* A "lifestyle business" (small "mom and pop" shop with no ambitions to grow beyond 1-2 locations)
* A public company
* A private company with $1-100M+ in revenue that is profitable but not growing YoY

I'm with Rachel that we should recognize that it's a word without a broadly agreed upon definition. Additionally, I think we should emphasize that startups are different from a lot of other businesses in that they not only need to attract *really stellar* talent in order to tackle a big problem quickly, *they need to incentivize people to come work for them when the business is cash poor.*

Reply:
The "what it's not" tact is very helpful! Nice work, @AndySparks